packages/mediacenter/kodi/patches/kodi-100.18-remove-ffi-and-intl-requirements.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/project/cmake/modules/FindPython.cmake b/project/cmake/modules/FindPython.cmake
```

Comment:
https://github.com/stefansaraev/xbmc/commit/cba220789264d9b424a311d3c05d3434fc9abe29 may be enough for you. too.